contributors/devel/api_changes.md

```diff
@@ -347,21 +348,25 @@ before starting "all the rest".
 
 ### Edit types.go
 
-The struct definitions for each API are in `pkg/api/<version>/types.go`. Edit
-those files to reflect the change you want to make. Note that all types and
-non-inline fields in versioned APIs must be preceded by descriptive comments -
-these are used to generate documentation. Comments for types should not contain
-the type name; API documentation is generated from these comments and end-users
-should not be exposed to golang type names.
+The struct definitions for each API are in
+`staging/src/k8s.io/api/<group>/<version>/types.go`. Edit those files to reflect
```

Comment:
https://github.com/kubernetes/kubernetes/pull/48746 made some changes which I believe require this line to be added to go struct of runtime Objects to generate `DeepCopyObject` functions in `<path_to_versioned_api>/zz_generated.deepcopy.go`:

```go
// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

@sttts